cmd/opdoc/opdoc.go

```diff
@@ -270,14 +270,19 @@ func typeString(types []logic.StackType) string {
 		case logic.StackAny:
 			out[i] = '.'
 		case logic.StackNone:
-			if i == 0 && len(types) == 1 {
-				return ""
-			}
-			panic("unexpected StackNone in opdoc typeString")
+			out[i] = '_'
 		default:
 			panic("unexpected type in opdoc typeString")
 		}
 	}
+
+	if strings.Contains(string(out), "_") {
```

Comment:
Adding the ecdsa curve immediate args, which have have a `StackNone` type, caused this to panic without a change like this.